pkg/state/state.go

```diff
@@ -289,6 +290,7 @@ type ReleaseSpec struct {
 	Values    []interface{}     `yaml:"values,omitempty"`
 	Secrets   []interface{}     `yaml:"secrets,omitempty"`
 	SetValues []SetValue        `yaml:"set,omitempty"`
+	Duration  time.Duration     `yaml:"duration,omitempty"`
```

Comment:
Sorry for the delay! "Soon" took 2 weeks...
Well anyway, this looks AWESOME except this line!

Presuming we won't decode the duration from YAML, can we perhaps change this to a private field, and omit the YAML field tag?

That way, it's clearer that the field is intended for taking note of the duration, instead of reading the value from YAML.

Reply:
@mumoshu good catch. I think so.

Reply:
I.e:

```suggestion
	duration  time.Duration
```

Right?

Reply:
yeah. @felipecrs

Reply:
As a maintainer, can you apply the suggestion above so we can check if tests still passes?

I'm a colleague of @AlexAxenti and I'm sure he won't mind. And he's probably not available until at least ~10 hours.